azkaban-db/src/main/sql/create.containerization-tables-all.sql

```diff
@@ -0,0 +1,71 @@
+CREATE TABLE IF NOT EXISTS image_types (
+  id               INT         NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  type             VARCHAR(64) NOT NULL UNIQUE,
+  description      VARCHAR(2048),
+  active           BOOLEAN,
+  deployable       VARCHAR(64),
+  created_on       TIMESTAMP       NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  created_by       VARCHAR(64)     NOT NULL,
+  modified_on      TIMESTAMP       DEFAULT NULL,
+  modified_by      VARCHAR(64)
+);
+
+CREATE TABLE IF NOT EXISTS image_versions (
+  id               INT         NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  path             VARCHAR(1024) NOT NULL,
+  description      VARCHAR(2048),
+  version          VARCHAR(64) NOT NULL,
+  type_id          INT NOT NULL, FOREIGN KEY(type_id) references image_types (id),
+  state            VARCHAR(64) NOT NULL,
+  release_tag      VARCHAR(64) NOT NULL,
+  created_on       TIMESTAMP       NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  created_by       VARCHAR(64)     NOT NULL,
+  modified_on      TIMESTAMP       DEFAULT NULL,
```

Comment:
Same as above. Default should be CURRENT_TIMESTAMP. Same comment for modified_by field.

Reply:
Done.